### PR TITLE
Skeleton backend

### PR DESCRIPTION
**Pull request by enriquezrene**
This PR has some code to accomplish the requirements of ticket #1 

In a nutshel:
- Basic setup
- Including chai and mocha for testing
- Basic routing using express, you can validate pointing your browser to http://localhost:3000/people
- Readme instruction to make the app work locally

### CODE REVIEW

File: bpm-people-api/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+ 
+const express = require('express');
```
Comment (Merd13) [inline bpm-people-api/app.js]: Could we use ES6 instead? Maybe the 'import' method.

Comment (fabricioflores) [reply bpm-people-api/app.js]: If you want to use ES6, you will use babel.
We (an old dev team) have a stack that maybe you can use.
https://github.com/Silverhill/coupon-backend
you can check the available scripts in package.json, and also the base libraries you could use, like nodemon for example.
Also, if you have a question, feel free to asking me.

File: bpm-people-api/app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const app = new express();
+const bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+ 
+require('./routes/peopleRoutes')(app);
+ 
+app.listen(3000, () => {
+    console.log('Server up!');
+});
```
Comment (fabricioflores) [inline bpm-people-api/app.js]: this final new line, you can manage simply with editorconfig.
https://editorconfig.org/
Actually, you can add an .editorconfig for all kind of projects in any language, and all your team will have the same file format, identation, tab, etc. 
And almost all the IDE or editors are compatibles with it (maybe it you use notepad not :-) )

File: bpm-people-api/app.js (hunk 1 above)
Comment (Merd13) [reply bpm-people-api/app.js]: I’m not sure if we need to use babel, according to this http://kangax.github.io/compat-table/es6/, in node version higher than 7 (latest stable is 10), 98% of es6 features are supported. And I think for backend of bpm-people, we don’t need to think about users that might not have appropriate version of Node. Does it make sense?

File: bpm-people-api/app.js (hunk 2 above)
Comment (enriquezrene) [reply bpm-people-api/app.js]: I took borrow you .editorconfi @fabricioflores :)

File: bpm-people-api/app.js (hunk 1 above)
Comment (enriquezrene) [reply bpm-people-api/app.js]: done @Merd13

**Pull request merged**